.github/workflows/.dependabot-issues.yml

```
@@ -0,0 +1,26 @@
+    branches:
+      - master
+    types: [ opened, reopened]
+
+jobs:
+  dependency-issue:
+      if: github.event.pull_request.user.login  == 'dependabot[bot]'
+      runs-on: ubuntu-latest
+      env:
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      steps:
+        - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)